Tests/Pair/RatioProvider/ExchangerAdapterRatioProviderTest.php

```diff
@@ -31,7 +32,9 @@ protected function getRatioProvider(): RatioProviderInterface
     public function testInvalidCurrencyCode(): void
     {
```

Comment:
Can you explain the changes in this file, I don't follow here? Again you know this better than me but trying to learn :)

Reply:
Due to type hinting, this check has become obsolete. It will throw now TypeError instead of MoneyException